ledger/participant-integration-api/src/main/scala/platform/apiserver/error/ErrorInterceptor.scala

```diff
@@ -54,24 +55,25 @@ final class ErrorInterceptor extends ServerInterceptor {
         * Knowing that Status.INTERNAL is used only by [[com.daml.error.ErrorCategory.SystemInternalAssumptionViolated]],
         * which is marked a security sensitive, we have the following heuristic: check whether gRPC status is Status.INTERNAL
         * and gRPC status description is not security sanitized.
+        * 2. Handling of Status.UNKNOWN:
+        * We do not have an error category that uses UNKNOWN so there is no risk of catching a legitimate Ledger API error code.
+        * A Status.UNKNOWN can arise when someone (us or a library):
```

Comment:
I'm a bit concerned of some internal gRPC implementation logic using Status UNKNOWN for some specific purpose but don't have any examples of what could go wrong when we intercept and process it. 
Should be benign I guess

Reply:
**UNKNOWN** = ```Unknown error. For example, this error may be returned when a Status value received from another address space belongs to an error space that is not known in this address space. Also errors raised by APIs that do not return enough error information may be converted to this error.``` (see https://grpc.github.io/grpc/core/md_doc_statuscodes.html)